Source/CarthageKit/Xcode.swift

```diff
@@ -169,6 +169,10 @@ public struct BuildArguments {
 			args += [ "BITCODE_GENERATION_MODE=\(bitcodeGenerationMode.rawValue)" ]
 		}
 
+		// Disable code signing requirement for all builds
+		// Frameworks get signed in the copy-frameworks action
+		args += [ "CODE_SIGNING_REQUIRED=NO" ]
```

Comment:
We need to add `CODE_SIGN_IDENTITY=` as an argument here. With that, I'm able to build Alamofire for all platforms without any certificates.